sympy/physics/optics/utils.py

```diff
@@ -460,15 +460,28 @@ def lens_makers_formula(n_lens, n_surr, r1, r2):
         Radius of curvature of first surface.
     r2 : sympifiable
         Radius of curvature of second surface.
+    d : sympifiable
+        Thickness of lens.
+        Applicable when lens = 2
+    lens : integer, default 0
+        0 for Thin Lens, 1 for plano lens,
+        2 for thick lens.
```

Comment:
It would be better to use a string like `lens="thick"`.

Having two separate argument `d` and `lens` for this is confusing. Maybe there's a better way of specifying his.

Reply:
Yes, that could be done